### PR TITLE
6.9 Add total story points suggestion

### PR DESCRIPTION
**Pull request by SherbaKirill**
image
image
Add total count of SP in sprint to View
Add average count of Sp in completed sprint to View

### CODE REVIEW

File: src/Teams.Web/Controllers/ManageSprintsController.cs [hunk 1]
```diff
@@ -171,6 +171,19 @@ public async Task<IActionResult> GetSprintById(int sprintId)
                     Completed = t.Completed
                 }
             ));
+            var totalSp = sprint.Tasks.Count>0 ? sprint.Tasks.Sum(t => t.StoryPoints) : 0;
```
Comment (SergeyMokin) [inline src/Teams.Web/Controllers/ManageSprintsController.cs:174]: spaces?

File: src/Teams.Web/Controllers/ManageSprintsController.cs [hunk 2]
```diff
@@ -171,6 +171,19 @@ public async Task<IActionResult> GetSprintById(int sprintId)
...
+            var sprints = await _manageSprintsService.GetAllSprintsAsync(sprint.TeamId, new DisplayOptions { });
```
Comment (SergeyMokin) [inline src/Teams.Web/Controllers/ManageSprintsController.cs:175]: other brackets? { } => ()

File: src/Teams.Web/ViewModels/Sprint/SprintViewModel.cs [hunk 3]
```diff
@@ -17,5 +17,6 @@ public class SprintViewModel
         public int Status { get; set; }
         public bool IsOwner { get; set; }
         public List<TaskViewModel> Tasks { get; set; }
+        public string TotalAverageSp { get; set; }
```
Comment (SergeyMokin) [inline src/Teams.Web/ViewModels/Sprint/SprintViewModel.cs:20]: Any benefits of using string?
Not fully clear naming

File: src/Teams.Web/Controllers/ManageSprintsController.cs [hunk 4]
```diff
@@ -171,6 +171,19 @@ public async Task<IActionResult> GetSprintById(int sprintId)
...
+            sprints = sprints.Where(t => t.Status == PossibleStatuses.CompletedStatus);
+            sprintViewModel.TotalAverageSp = totalSp.ToString();
+
+            if (sprints.Count()>0)
```
Comment (SergeyMokin) [inline src/Teams.Web/Controllers/ManageSprintsController.cs:179]: please move calculation to service layer

Comment (SergeyMokin) [reply src/Teams.Web/Controllers/ManageSprintsController.cs:179]: business logic should be not in this layer.

**Review by SergeyMokin — CHANGES_REQUESTED** [review]

File: src/Teams.Web/Controllers/ManageSprintsController.cs [hunk 5]
```diff
@@ -171,6 +171,8 @@ public async Task<IActionResult> GetSprintById(int sprintId)
                     Completed = t.Completed
                 }
             ));
+            sprintViewModel.TotalStoryPoint = sprint.Tasks.Count > 0 ? sprint.Tasks.Sum(t => t.StoryPoints) : 0;
```
Comment (SergeyMokin) [inline src/Teams.Web/Controllers/ManageSprintsController.cs:174]: in current

File: src/Teams.Web/Controllers/ManageSprintsController.cs [hunk 6]
```diff
@@ -171,6 +171,8 @@ public async Task<IActionResult> GetSprintById(int sprintId)
...
+            sprintViewModel.AverageStoryPoint = await _manageSprintsService.GetAverageStoryPointAsync(sprint);
```
Comment (SergeyMokin) [inline src/Teams.Web/Controllers/ManageSprintsController.cs:175]: in completed

**Review by SergeyMokin — APPROVED** [review]

**Review by sub13 — APPROVED** [review]

**Review by Y-floryanovich — APPROVED** [review]

**Pull request merged**